.github/workflows/build.yml

```diff
@@ -15,20 +15,25 @@ jobs:
     - uses: actions/checkout@v3
     - name: dependencies
       run: |
-        sudo apt update
-        sudo apt install libgtest-dev libgflags-dev openssl libssl-dev protobuf-compiler
+        sudo apt update -y
+        sudo apt install -y libgtest-dev libgflags-dev openssl libssl-dev protobuf-compiler protoc-gen-go golang-go
```

Comment:
Please check & confirm that I am naming the right s/w components to be installed. I got this list off of INSTALL.md doc.